Source/CarthageKit/Xcode.swift

```diff
@@ -83,106 +83,6 @@ extension ProjectLocator: CustomStringConvertible {
 	}
 }
 
-/// Configures a build with Xcode.
-public struct BuildArguments {
```

Comment:
It's fine now, but next time can you try to keep things like this in separate commits? That way it's easier to review, since I can look at the moved code in one commit, and tests in another. It also makes potentially needing to revert one or the other easier.